controllers/awsmachine_controller.go

```diff
@@ -448,57 +448,52 @@ func (r *AWSMachineReconciler) getOrCreate(scope *scope.MachineScope, ec2svc ser
 		return nil, err
 	}
 
-	if instance == nil {
-		scope.Info("Creating EC2 instance")
-		// Create a new AWSMachine instance if we couldn't find a running instance.
+	if instance != nil {
+		return instance, nil
+	}
+	scope.Info("Creating EC2 instance")
+
```

Comment:
Add back the comment? // Create a new AWSMachine instance if we couldn't find a running instance.